example.init.yaml

```diff
@@ -242,3 +242,6 @@ enable_dockerfile_lang: false
 # Set to true to enable scaling to zero
 scale_to_zero: false
 openfaas_cloud_version: 0.9.0
```

Comment:
This must be updated to 0.9.1 otherwise the clone won't find valid network policy files.

https://github.com/openfaas/openfaas-cloud/releases/tag/0.9.1

Reply:
Will do 👍